plugin.program.newgrounds/addon.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.program.newgrounds" name="Newgrounds" version="1.0.5" provider-name="New Age Software LLC">
+    <requires>
+		<import addon="xbmc.python" version="2.25.0"/>
+		<import addon="script.module.beautifulsoup" version="3.2.1"/> 
+	</requires>
+	<extension point="xbmc.python.pluginsource" library="addon.py">
+		<provides>executable</provides>
+	</extension>
+	<extension point="xbmc.addon.metadata">
+		<platform>all</platform>
+		<summary lang="en">newgrounds</summary>
+		<description lang="en">Plugin to newgrounds.com content</description>
+		<license>GNU General Public License, v2</license>
```

Comment:
Please use a spdx license identifier instead of the full license name if possible. See: https://spdx.org/licenses/